### PR TITLE
Updated to use cf-core and new docs template and adds dynamic animation duration based on content height

### PR DESCRIPTION
**Pull request by himedlooff**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-expandables",
-  "version": "0.3.5",
+  "version": "1.4.0",
```
Comment (Scotchester) [inline bower.json]: **1**.4?

Comment (himedlooff) [inline bower.json]: The update adds a new feature.

Comment (Scotchester) [inline bower.json]: Does it break back-compat, though?

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 1. MAJOR version when you make incompatible API changes,
> 2. MINOR version when you add functionality in a backwards-compatible manner, and
> 3. PATCH version when you make backwards-compatible bug fixes.

Comment (himedlooff) [inline bower.json]: Nope.

Comment (Scotchester) [inline bower.json]: So we should just go to 0.4.0 for now.

Comment (himedlooff) [inline bower.json]: Haaa sorry, didn't realize I changed the major part... Whoops!

**Comment by himedlooff** [comment]
Fixed!

File: demo/static/js/component.js [hunk 2]
```diff
@@ -10519,49 +10519,65 @@ jQuery.extend( jQuery.easing,
...
 
       });
 
     });
 
-  });
+  };
+
+  $.fn.expandable.calculateExpandDuration = function( height ) {
+    return height * 4;
```
Comment (Scotchester) [inline demo/static/js/component.js]: Should we put a max on these? Expandables can potentially be quite tall. At 500px tall, we're waiting 2 full seconds.

Comment (himedlooff) [inline demo/static/js/component.js]: Fixed!

File: src/less/cf-expandables.less [hunk 3]
```diff
@@ -5,22 +5,83 @@
 
 
 /* topdoc
+    name: Recommended expandable pattern
+    family: cf-expandables
+    notes:
+      - "Expandables can be built by combing the basic barebones structure
```
Comment (Scotchester) [inline src/less/cf-expandables.less:11]: combining?

**Comment by Scotchester** [comment]
Alrighty! Need to revisit some of the classing at some point (see #30), but this is good for now.

**Pull request merged**